Snake/BusinessLayer/GameSound.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Media;
+using System.IO;
+
+namespace BusinessLayer
+{
+    class GameSound
+    {
+        private string path = System.IO.Directory.GetCurrentDirectory();
```

Comment:
Don't use fully qualified names unless there is an ambiguity. Just Directory.GetCurrentDirectory() should be fine.